practices/handover.md

```diff
@@ -0,0 +1,72 @@
+
+In a broad sense, the handover procedure involves the following steps:
+1. Identify the team/individual that will be on-boarding onto the project
+2. Sync up schedules and timings with the said team/individual in order to find slots that work for both teams
+3. Set-up a recurring meeting in the identified slots
+4. Have a clear stratergy on the key aspects of the projects that you aim to cover
+5. Always leave room for questions, follow-ups and doubts!
+
+## Handover Stratergy
+
+Before initiating the handover process, the current team managing the project must have a clear vision about the end goal of this transition. That is, it is in the best interests of both teams to identify the key outcomes of this process.
+
+For example: At the end of the subsequent handover, the team/individual should be hands-on with the project. They should be able to work on XYZ functionalities, answer any questions raised by the stakeholders and know the finer workings of the project from both a technical as well as business standpoint.
+
+A typical handover stratergy would cover the following aspects:
+1. The delivery method/channel for the knowledge transfer
+2. Identified SPOCs from the current team as well as from the onboarding team that will be participating
+3. How often the communication will occur? (Daily, Weekly, etc.)
+
+## Best Practicses
+
+1. __Starting Early:__ Initiating the process of handing over a project earlier gives both the receiving team and the initiating team ample time to cross-communicate and resolve any doubts or misunderstandings about the project. It gives the initiating team time to gain confidance that the project is in good hands, and the receiving team enough time to know the project through and through.
+
+2. __Involving The Operations Team:__ Involving a SPOC from the support team enables the building of relationships between the new individual/team and the operations team, who is usually responsible for the deployment of the project. This also enables a two-way communication, allowing the successors of the project to know about any added requirements/protocols from the support side as well.
+
+3. __Update All Relevant Documentation Frequently:__ This is a predecessor to the handover project, and is the responsibility of the current team managing the project. Keeping project documentation updated not only reaffirms the understanding of the project on behalf of the current team, but also makes it easier for the next team to quickly adapt to the project.
+
+4. __Multi-faceted Knowledge Transfer Process:__ Knowledge transfer during handover can take place across multiple channels. While face-to-face communication such as over a meeting online or in real-life is the most common, channels such as video guides, resource lists, workarounds and the sharing of previous obstacles and their corresponding solutions aid the process multifold.
+
```

Comment:
Could you, please, add pitfalls of handover? What are the common things that go wrong (or sour)?

Add them into a separate part:
## Common Pitfalls of Handover
- Pitfall one (Lack of communication between involved parties)
- Not enough time for proper handover
- Something else